models/intel/resnet50-int8-sparse-v2-onnx-0001/description/resnet50-int8-sparse-v2-onnx-0001.md

```diff
@@ -0,0 +1,52 @@
+# resnet50-int8-sparse-v2-onnx-0001
```

Comment:
What does `v2` mean? Is this information crucial? What is the difference with `v1` in the model below?

Reply:
@eizamaliev v2 is version of sparsfication algorithm, do not mix it with version of topology